src/App.tsx

```diff
@@ -1,13 +1,60 @@
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-no-comment-textnodes */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+type Field = 'all' | 'active' | 'completed';
+
+const USER_ID = 11140;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // if (!USER_ID) {
+  //   return <UserWarning />;
+  // }
+  const [todos, setTodos] = useState([] as Todo[]);
+  const [initialTodos, setInitialTodos] = useState([] as Todo[]);
+  const [errorMessage, setMessage] = useState('');
+  const [selectedField, setSelectedField] = useState<Field>('all');
+  const [isErrorUnnount, setIsErrorUnnount] = useState(false);
+  const [isDblClick, setIsDblClick] = useState(false);
+
+  const timer = () => {
+    setTimeout(() => {
+      setIsErrorUnnount(true);
+    }, 3000);
+  };
+
+  timer();
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(someTodos => {
+        setTodos(someTodos);
+        setInitialTodos(someTodos);
+      })
+      .catch(() => setMessage('Incorrect URL'));
+    // .finally(() => setTimeout(() => setMessage(false), 3000));
+  }, []);
+
+  let preparedTodos = [...todos];
+
+  useMemo(() => {
```

Comment:
`useMemo` hook returns calculated value
```suggestion
  const preparedTodos = useMemo(() => {
```